content/en/docs/languages/java/instrumentation.md

```diff
@@ -42,6 +42,15 @@ manual instrumentation.
 You don't have to use the example app: if you want to instrument your own app or
 library, follow the instructions here to adapt the process to your own code.
 
+### Prerequisites
+
+For running the example app, ensure that you have the following installed
+locally:
+
+- Java JDK 17+, due to the use of Spring Boot. OpenTelemetry Java only [requires
+  Java 8+][java-vers] otherwise.
+- [Gradle](https://gradle.org/) or [Maven](https://maven.apache.org/)
```

Comment:
The example app only has a gradle configuration. Its possible translate to the equivalent maven, but that doesn't exist right now.

Reply:
good call out, thanks!

Reply:
@pegasas please take a look at this open conversation

Reply:
@pegasas can you take another look here? The simple solution is probably to drop the requirement for Maven, although the preferred solution would to have both